commands/push_test.go

```diff
@@ -1,21 +1,10 @@
 package commands
 
 import (
-	"github.com/bmizerany/assert"
+	"github.com/github/hub/internal/assert"
 	"testing"
 )
 
-func testPush(t *testing.T) {
```

Comment:
Any reason why this test was deleted?

Reply:
This func was not starting with capital T so I thought it is a test helper. And it was not used from anywhere.